UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -93,5 +93,30 @@ public async Task WhenRequestUploadFileShareServiceData_ThenReturnsCloudBlockBlo
 
             Assert.AreEqual("http://tempuri.org/blob", result);
         }
+
+        [Test]
+        public void WhenRequestDownloadFileShareServiceFiles_ThenReturnsFileTospecificPath()
+        {
+            var message = new SalesCatalogueServiceResponseQueueMessage() { 
+                BatchId = "63d38bde-5191-4a59-82d5-aa22ca1cc6dc"
+            };
+            var fulfillmentDataResponses = new List<FulfillmentDataResponse>() {
+                new FulfillmentDataResponse{ BatchId = "63d38bde-5191-4a59-82d5-aa22ca1cc6dc", EditionNumber = 10, ProductName = "Demo", UpdateNumber = 3, FileUri = new List<string>{ "http://ffs-demo.azurewebsites.net" } }
+            };
+            var result = fulfilmentFileShareService.DownloadFileShareServiceFiles(message, fulfillmentDataResponses, "");
+            Assert.IsNotNull(result);
+        }
+
+        [Test]
+        public void WhenRequestDownloadFileShareServiceFiles_ThenReturnsNoFileTospecificPath()
+        {
+            var message = new SalesCatalogueServiceResponseQueueMessage()
+            {
+                BatchId = "63d38bde-5191-4a59-82d5-aa22ca1cc6dc"
+            };
+            var fulfillmentDataResponses = new List<FulfillmentDataResponse>();
```

Comment:
The variable may need updating if fulfillment is updated elsewhere

Reply:
updated variable

Reply:
renamed